test.js

```diff
@@ -41,6 +44,78 @@ test('Prints `--help`', (assert) => {
 
 
 test((
+  'Falls back to plain text `--help` over stdout'
+), (assert) => new Promise((resolve, reject) => {
+  assert.plan(3);
+  setTimeout(reject, 100);
+
+  const expectedHelpContent = (
+`I’m dreaming
+of a white Christmas
+`
+  );
+  fs.writeFileSync(`${__dirname}/manpages/elm-live.1.txt`, expectedHelpContent);
+
+  const crossSpawn = { sync: (command) => {
+    assert.is(command, 'man',
+      'tries to spawn `man`'
+    );
+
+    return { error: { code: 'ENOENT' } };
+  } };
+
+  const outputStream = qs((chunk) => {
+    assert.is(chunk, expectedHelpContent,
+      'prints elm-make output first'
```

Comment:
Yup, if I understand you well. Help output should be the same as the contents of `/manpages/elm-live.1.txt`.